experimental/list_formatter/src/deduplicating_array.rs

```diff
@@ -0,0 +1,111 @@
+use core::{mem, ops::Index, ptr};
```

Comment:
Is it expected that the postcard file is +3.69 KB with this change?  We should be able to make it a no-op, right?

Reply:
I don't understand where this is coming from. One thing I removed was the default fallback, but the ", " ListJoinerPattern should only be 4 bytes (1 string length, 2 string, 1 index), and because of deduping that should only appear once per language. That said, I'm removing the discriminants of the Optionals, so that should be a 12 byte decrease per language.